CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
+### [1.0.3] 
+- Upgraded to Angular 6
```

Comment:
Angular 6 should be 1.2.0, there is ongoing 1.1.0 fixes https://github.com/ng2-ui/auto-complete/milestone/1

Reply:
Done.